src/community/wfs3/pom.xml

```diff
@@ -145,6 +145,43 @@
       <artifactId>jsoup</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.geotools</groupId>
+      <artifactId>gt-app-schema</artifactId>
+      <version>${gt.version}</version>
+    </dependency>
```

Comment:
So WFS 3 depends on App-Schema now? Just making sure this is not a forgotten dependency.

Reply:
Needed to run a request based on app-schema... ah, but you're right, could have been a test dependency. Fixing.